modules/postgres/README.md

```diff
@@ -180,3 +180,7 @@ should give you clues as to why the collector isn't working.
   ```bash
   ./go.d.plugin -d -m postgres
   ```
+  
+## Additional information
+  
+- Read our blog post on [PostgreSQL monitoring with Netdata](https://www.netdata.cloud/blog/postgresql-monitoring)
```

Comment:
I think we can put that in the very first section. What do you think?

Reply:
Yes - I was debating whether to put it at the start or the end.